src/main/java/minicraft/saveload/Load.java

```diff
@@ -621,11 +621,6 @@ public void loadInventory(Inventory inventory, List<String> data) {
 				continue;
 			}
 			
-
-			if (worldVer.compareTo(new Version("2.0.7-dev1")) < 0) {
-				item = subOldName(item, worldVer);
-			}
-			
 			if (worldVer.compareTo(new Version("2.1.0-dev1")) < 0) {
 				item = subOldName(item, worldVer);
 			}
```

Comment:
I don't understand why you removed the upper one; please undo and remove the bottom one instead.

Reply:
ok

Reply:
but i warn that it may not work i tried it before

Reply:
My bad